frontend/app/src/form/index.tsx

```diff
@@ -329,163 +391,242 @@ export default function Form(props: any) {
             ) : props?.newDesign ? (
               <>
                 <CreateBountyHeaderContainer color={color}>
-                  <EuiText className="stepText">{`STEP ${schemaData.step}/3`}</EuiText>
-                  <EuiText className="HeadingText">{schemaData.heading}</EuiText>
-                  {schemaData.sub_heading && (
-                    <EuiText
-                      className="SubHeadingText"
-                      style={{
-                        marginBottom: schemaData.step === 1 ? '29px' : '37px'
-                      }}
-                    >
-                      {schemaData.sub_heading}
+                  <div className="TopContainer">
+                    <EuiText className="stepText">
+                      {`STEP ${schemaData.step}`} <span className="stepTextSpan"> / 5</span>
```

Comment:
we can make that a .length value if we change BountyDetailsCreationData to be an array